ESP8266/ESP8266.cpp

```diff
@@ -215,7 +220,7 @@ void ESP8266::_packet_handler()
     uint32_t amount;
 
     // parse out the packet
-    if (!_parser.recv(",%d,%d:", &id, &amount)) {
+    if (!_parser.recv(",%d,%" SCNu32 ":" , &id, &amount)) {
```

Comment:
If we want to be pedantic, we should change `ESP8266::send` to take in `size_t`. I'm not sure SCNu32 adds anything useful here.